### PR TITLE
DB Access in UT 

### PR DESCRIPTION
**Pull request by lucasborin**
It acts differently according to the RISK LEVEL classification of the test class under evaluation. 
- RISK LEVEL HARMLESS or missing RISK LEVEL classification --> Any DB Operation will be reported;
- RISK LEVEL DANGEROUS/CRITICAL --> UPDATE and/or DELETE operation(s) will be reported.

### CODE REVIEW

**Comment by lucasborin** [comment]
I won't implement the abapLint's request because it will force the use of parentheses in the first `IF` and it will negatively impact the code readability.

File: src/checks/y_check_db_access_in_ut.clas.abap [hunk 1]
```diff
@@ -29,120 +57,152 @@ CLASS Y_CHECK_DB_ACCESS_IN_UT IMPLEMENTATION.
...
-
-      check_if_error( index = index
-                      statement = statement ).
-    ENDIF.
-  ENDMETHOD.
 
+    DATA(source_code) = consolidade_tokens( statement ).
 
-  METHOD is_persistent_object.
+    DATA(tokens) = COND #( WHEN test_risk_level = risk_level_harmless THEN tokens_not_allowed-for_harmless
```
Comment (lucasborin) [inline src/checks/y_check_db_access_in_ut.clas.abap:105]: it is not a good name, let's change it.

**Comment by larshp** [comment]
okay, then https://github.com/SAP/code-pal-for-abap/blob/db-access-in-ut/abaplint.json#L132 can be disabled

personally I prefer less statements, and think less statements makes the code more readable

**Comment by lucasborin** [comment]
I agree with you. Let me split the statement into more methods.

**Pull request merged**